modules/picard_qc/1.0/config/default.yaml

```diff
@@ -0,0 +1,58 @@
+lcr-modules:
+    
+    picard_qc:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "__UPDATE__"
+            intervals : "__UPDATE__"
+
+            # by default are generate by reference_files subworkflow
+            rrna_intervals: "genomes/{genome_build}/rrna_intervals/rRNA_int_gencode-33.txt"
+            refFlat: "genomes/{genome_build}/annotations/refFlat_gencode-33.txt"
```

Comment:
For the `star` module, I decided against including these in the config file. Because these files are tied to the `reference_files` workflow, they can't really be changed, so there's no point in including the paths here. In fact, I'm afraid the user will try to change them without realizing that they're wrapping in `reference_files()` in the Snakefile. Instead, you can use `reference_params` like I did in `star`.

Reply:
okay sounds good, I wasn't sure if users are allowed to modify those